src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handleClick = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextClick = () => {
+    if (currentPage < totalPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  const renderPageLinks = () => {
+    const links = [];
+
+    for (let i = 1; i <= totalPages; i++) {
+      links.push(
+        <li
+          className={cn('page-item', {
+            active: currentPage === i,
+          })}
+          key={i}
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${i}`}
+            onClick={() => onPageChange(i)}
+          >
+            {i}
+          </a>
+        </li>,
+      );
+    }
+
+    return links;
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+            disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
```suggestion
            aria-disabled={currentPage === 1}
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**